src/components/material/player-modal/PlayerModal.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import Modal from "../../../core/utils/modal";
+import { useText } from "../../../core/utils/text";
+import Player, { PlayerType } from "../../reader-player/Player";
+import { playerModalId } from "./helper";
+
+const PlayerModal = ({ identifier, orderId }: PlayerType) => {
+  const t = useText();
+
+  return (
+    <Modal
+      dataCy="player-modal"
+      // Todo: Find a better way to generate the modalId
+      modalId={playerModalId(identifier || orderId || "")}
+      screenReaderModalDescriptionText={t("playerModalDescriptionText")}
+      closeModalAriaLabelText={t("playerModalCloseButtonText")}
+    >
+      <Player identifier={identifier} />
```

Comment:
Can we center the player inside modal pls?
![image](https://github.com/user-attachments/assets/26432c45-f5a9-4a20-9ec9-7c95ee6d2a39)

Reply:
👍